docs/src/main/sphinx/object-storage/file-system-s3.md

```diff
@@ -42,11 +42,13 @@ support:
 * - `s3.sse.type`
   - Set the type of S3 server-side encryption (SSE) to use. Defaults to `NONE`
     for no encryption. Other valid values are `S3` for encryption by S3 managed
-    keys, and `KMS` for encryption with a key from the AWS Key Management
-    Service (KMS). Note that S3 automatically uses SSE so `NONE` and `S3` are
+    keys, `KMS` for encryption with a key from the AWS Key Management
+    Service (KMS), and `CUSTOMER` for encryption with customer-provided key. Note that S3 automatically uses SSE so `NONE` and `S3` are
     equivalent. S3-compatible systems might behave differently.
 * - `s3.sse.kms-key-id`
   - The identifier of a key in KMS to use for SSE.
+* - `s3.sse.customer-key`
+  - Customer Key to use in SSE-C mode for SSE.
```

Comment:
Why the capitalization "Customer Key" .. is that something specifically from AWS .. can we link to the docs for that. Or otherwise can we add more info on what this property needs .. is it a path to a file or an actual value? Can we add an example?